src/libs/utils.ts

```diff
@@ -116,20 +105,11 @@ export const memoizeAsync = (asyncFn, { keyFn = _.identity, expires = Infinity }
   };
 };
 
-export const withDelay = _.curry((ms, wrappedFn) => async (...args) => {
```

Comment:
This function was used once in ajax-common, but that code is clearer using `delay` directly instead of `withDelay`.